readme.txt

```diff
@@ -8,23 +8,24 @@ Stable tag: trunk
 Enable Two-Factor Authentication using time-based one-time passwords (OTP, Google Authenticator), Universal 2nd Factor (FIDO U2F, YubiKey), email and backup verification codes.
 
 == Description ==
-
-Use the "Two-Factor Options" section under "Users" → "Your Profile" to enable and configure one or multiple two-factor authentication providers for your account:
-
-- Email codes
-- Time Based One-Time Passwords (TOTP)
-- FIDO Universal 2nd Factor (U2F)
-- Backup Codes
-- Dummy Method (only for testing purposes)
+
+Use the "Two-Factor Options" section under "Users" → "Your Profile" to enable and configure one or multiple two-factor authentication providers for your account:
```

Comment:
What are the changes in this section of the diff? Is that the line endings or extra whitespace? Those should match the definitions in `.editorconfig`:

https://github.com/WordPress/two-factor/blob/ab8d6510a17606b200d1e44a603c13b5014bac52/.editorconfig